src/main/java/org/opentripplanner/transit/raptor/api/path/Path.java

```diff
@@ -201,11 +201,12 @@ public int hashCode() {
     private static <S extends RaptorTripSchedule> int countNumberOfTransfers(AccessPathLeg<S> accessLeg) {
         // Skip first transit
         PathLeg<S> leg = accessLeg.nextLeg().nextLeg();
-        int i = 0;
+        int i = accessLeg.access().numberOfLegs() - 1;
```

Comment:
Came up in discussion: this does not work in the general case with multiple sub-legs.